libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.module.spec.ts

```diff
@@ -21,7 +21,7 @@ import { render, screen } from '@testing-library/angular';
 import userEvent from '@testing-library/user-event';
 
 // eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
-import { DhConfigurationLocalizationModule } from '@energinet-datahub/dh/globalization/configuration-localization';
+import { translocoProviders } from '@energinet-datahub/dh/globalization/configuration-localization';
```

Comment:
suggestion: `danishLocalProviders` needs to be provided here, as it was before, not `TranslocoModule`.